Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -1054,7 +1054,10 @@ class ScrollView extends React.Component<Props, State> {
     |},
     animated?: boolean, // deprecated, put this inside the rect argument instead
   ) => {
-    invariant(Platform.OS === 'ios', 'zoomToRect is not implemented');
+    invariant(
+      Platform.OS === 'ios' || Platform.OS === 'macos',
+      'zoomToRect is not implemented',
+    ); // TODO [(macOS GH#774)
```

Comment:
Nit: starting and closing tag (since this is now multiline).

Reply:
Let's do it!